api/app/responses/api.py

```diff
@@ -146,8 +147,8 @@ def post(self):
         user = user_repository.get_by_id(user_id)
         responses = response_repository.get_all_for_user_application(user_id, application_form_id)
 
-        if not application_form.nominations and len(responses) > 0:
-            return errors.RESPONSE_ALREADY_SUBMITTED
+        # if not application_form.nominations and len(responses) > 0:
```

Comment:
I think we still need this check - some events and application forms should still only support a single response and some should support multiple responses and we should validate this is the case. How about changing the name `nominations` to `allows_many_responses` to make it more generic? Then default `allows_many_responses` to True for event-type journal